integration_tests/integration/serviceProviderReferrals.cy.js

```diff
@@ -2378,9 +2379,13 @@ describe('Service provider referrals dashboard', () => {
           cy.get('#duration-hours').should('have.value', '1')
           cy.get('#duration-minutes').should('have.value', '15')
 
-          cy.get('#date-day').clear().type('10')
-          cy.get('#date-month').clear().type('11')
-          cy.get('#date-year').clear().type('2022')
+          cy.get('#date-day')
+            .clear()
+            .type((today.getDate() + 1).toString())
+          cy.get('#date-month')
+            .clear()
+            .type((today.getMonth() + 1).toString())
+          cy.get('#date-year').clear().type(today.getFullYear().toString())
```

Comment:
_note_
This will likely have edge cases around the last day of the month and the last month of the year.

Reply:
We could use [momentjs](https://momentjs.com/) to express things like `moment().add(10, 'days')`

Reply:
Yes David, I noticed that after committing,I will fix this as part of other fix